application/src/main/java/uk/nhs/hee/tis/revalidation/changelog/TisStatusCorrectionChangeLog.java

```diff
@@ -40,15 +37,14 @@
 @Slf4j
 public class TisStatusCorrectionChangeLog {
 
-    @ChangeSet(order = "001", id = "insertCorrectTisStatuses", author = "", runAlways = true)
+    @ChangeSet(order = "001", id = "insertCorrectTisStatuses", author = "")
```

Comment:
I'd prefer us to be more definitive about the order that the changesets run in.

Reply:
Do you mean by adding an order to the ChangeLog annotation?

Reply:
Yep.  From a bit of digging it looks like the order is only effective within a single `@Changlog` so ensuring ordering would require a larger refactor.  We should take a look at a larger refactor (separately) as there are other Mongock best practices that we should consider.  https://mongock.io/v5/migration/index.html#best-practices

Reply:
That's a good read, thanks!